enterprise/backend/src/metabase_enterprise/metabot_v3/tools/util.clj

```diff
@@ -26,3 +28,12 @@
        :name (get column :display-name)
        :type (convert-field-type column)}
       (m/assoc-some :description (get column :description))))
+
+(defn get-table
+  "Get the table for `id`."
+  [id]
+  (when-let [table (t2/select-one [:model/Table :id :name :description :db_id] id)]
+    (when (mi/can-read? table)
+      (-> table
+          (t2/hydrate :metrics)
+          (assoc :id id)))))
```

Comment:
Why do we need to assoc the id here? Isn't it already selected?

Reply:
I've simplified/optimized the thing.